client/src/components/teams/TeamDashboard.vue

```diff
@@ -0,0 +1,249 @@
+<template>
+  <v-container>
+    <v-row class="mx-6" :fullscreen="mobile">
+      <!-- Team Name -->
+      <v-row class="my-2">
+        <v-col align="center">
+          <h1>{{ teamData.team_name }}</h1>
+        </v-col>
+      </v-row>
+      <v-divider></v-divider>
+
+      <!-- Total Kilometres -->
+      <v-row align="center" class="my-2">
+        <v-icon class="mdi mdi-run-fast ml-3" size="50px"></v-icon>
+        <v-col>
+          <h2 class="totalKms">{{ teamData.total_kilometres }} KM</h2>
+          <h3>TOTAL</h3>
+        </v-col>
+      </v-row>
+      <v-divider></v-divider>
+
+      <!-- Invite Code -->
+      <v-row align="center" class="my-2">
+        <v-col>
+          <h2>Invite Code</h2>
+          <p class="invite-code">{{ teamData.invite_code }}</p>
+        </v-col>
+        <v-col align="end" class="px-5">
+          <v-icon
+            @click="copyInviteCode"
+            class="mdi mdi-clipboard-multiple-outline"
+            size="32px"
+          ></v-icon>
+        </v-col>
+      </v-row>
+      <v-divider></v-divider>
+
+      <!-- Bio -->
+      <v-row id="pointer-cursor" class="my-2">
+        <v-col @click="toggleBio">
+          <h2>Bio</h2>
+          <p v-show="isBioVisible">{{ teamData.bio }}</p>
+        </v-col>
+        <v-col align="end" class="px-3">
+          <v-icon
+            v-if="user.is_admin"
+            icon="mdi mdi-pencil"
+            size="32px"
+            @click="editBioDialog = true"
+            class="px-6"
+          ></v-icon>
+          <v-dialog v-model="editBioDialog" width="auto">
+            <v-card>
+              <v-card-text>
+                What would you like to set the {{ teamData.team_name }} bio to?
+              </v-card-text>
+              <v-textarea v-model="newBioText" placeholder="Bio"></v-textarea>
+              <v-card-actions>
+                <v-btn @click="editBio">Submit</v-btn>
+                <v-btn @click="editBioDialog = false">Cancel</v-btn>
+              </v-card-actions>
+            </v-card>
+          </v-dialog>
```

Comment:
I would make a component for the edit bio. Actually it might be better to move the edit utton next to the team name so that you can edit both the team name and bio.

<img width="251" alt="Screen Shot 2023-07-04 at 9 01 29 pm" src="https://github.com/codersforcauses/csf/assets/45170409/83635f30-ba4d-44e6-9ba8-a093b8b61a89">